src/extractor/guidance/turn_lane_augmentation.cpp

```diff
@@ -107,20 +106,65 @@ LaneDataVector augmentMultiple(const std::size_t none_index,
         util::Log(logWARNING) << "Failed lane assignment. Reached bad situation.";
         return std::make_pair(std::size_t{0}, std::size_t{0});
     }();
-    for (auto intersection_index = range.first; intersection_index < range.second;
-         ++intersection_index)
+
+    const auto allowed_in_range =
```

Comment:
In the following, we need intersection.begin() + range.first/second a lot. Want to make temp. vars for them?